index.d.ts

```diff
@@ -1,13 +0,0 @@
-export default class TaskListsElement extends HTMLElement {
-  disabled: boolean
-  sortable: boolean
-}
-
-declare global {
-  interface Window {
-    TaskListsElement: typeof TaskListsElement
-  }
-  interface HTMLElementTagNameMap {
-    'task-lists': TaskListsElement
```

Comment:
@koddsson do you have context on this addition? Looks like by default tsc does not generate these lines.

Reply:
Yup. Context is [here](https://github.com/github/image-crop-element/pull/26).

Reply:
Added them in https://github.com/github/task-lists-element/pull/16/commits/641ba8f247424a400ec1c7e921b9e0dfef965440.